consensus/polybft/fsm.go

```diff
@@ -373,25 +367,23 @@ func (f *fsm) Validate(proposal []byte) error {
 		return err
 	}
 
-	if extra.Checkpoint == nil {
-		return fmt.Errorf("checkpoint data for block %d is missing", block.Number())
+	if extra.BlockMetaData == nil {
+		return fmt.Errorf("block meta data for block %d is missing", block.Number())
 	}
 
-	if parentExtra.Checkpoint == nil {
-		return fmt.Errorf("checkpoint data for parent block %d is missing", f.parent.Number)
+	if parentExtra.BlockMetaData == nil {
+		return fmt.Errorf("block meta data for parent block %d is missing", f.parent.Number)
 	}
 
 	if err := extra.ValidateParentSignatures(block.Number(), f.polybftBackend, nil, f.parent, parentExtra,
-		f.backend.GetChainID(), signer.DomainCheckpointManager, f.logger); err != nil {
+		f.backend.GetChainID(), signer.DomainBridge, f.logger); err != nil {
```

Comment:
`chainID` is unused, so can be removed from `ValidateParentSignatures`.
```suggestion
		signer.DomainBridge, f.logger); err != nil {
```